common/src/com/intellij/plugins/haxe/module/impl/HaxeModuleSettingsBaseImpl.java

```diff
@@ -200,19 +200,19 @@ public void setHxmlPath(String hxmlPath) {
   }
 
   public boolean isUseHxmlToBuild() {
-    return buildConfig == USE_HXML;
+    return HaxeConfiguration.translateBuildConfig(buildConfig) == HaxeConfiguration.HXML;
```

Comment:
Could you make property to reduce code duplication? There is a 5 similar places in this diff.

Reply:
What are you expecting?  `return HaxeConfiguration.HXML.asBuildConfig() == buildConfig;`?  Or do you have something else in mind?

Reply:
Or, do you mean a _method_ instead of _property_?

Reply:
Yes, method:
```java
private boolean isConfigurationKindOf(HaxeConfiguration config) {
   return HaxeConfiguration.translateBuildConfig(buildConfig) == config;
}
```